pkg/toolsets/kubevirt/vm/create/plan.tmpl

```diff
@@ -0,0 +1,99 @@
+# VirtualMachine Creation Plan
+
+**IMPORTANT**: Always use `runStrategy` instead of the deprecated `running` field when creating VirtualMachines.
+
+Use the `resources_create_or_update` tool with the following YAML:
+
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachine
+metadata:
+  name: {{.Name}}
+  namespace: {{.Namespace}}
+spec:
+  runStrategy: Halted
+{{- if .Instancetype}}
+  instancetype:
+    name: {{.Instancetype}}
+    kind: VirtualMachineClusterInstancetype
+{{- end}}
+{{- if .Preference}}
+  preference:
+    name: {{.Preference}}
+    kind: VirtualMachineClusterPreference
+{{- end}}
+{{- if .UseDataSource}}
+  dataVolumeTemplates:
+  - metadata:
+      name: {{.Name}}-rootdisk
+    spec:
+      sourceRef:
+        kind: DataSource
+        name: {{.DataSourceName}}
+        namespace: {{.DataSourceNamespace}}
+      storage:
+        resources:
+          requests:
+            storage: 30Gi
+{{- end}}
+  template:
+    spec:
+      domain:
+        devices:
+          disks:
+          - name: {{.Name}}-rootdisk
+{{- if not .Instancetype}}
+        memory:
+          guest: 1Gi
```

Comment:
Would it make sense to set more memory here? The only OS according to common preferences requiring 1Gi is RHEL7, other OSes require more memory.